login/style.css

```diff
@@ -1,4 +1,73 @@
-#login-image {
-  width: 100px;
-  height: 100px;
-}
\ No newline at end of file
+
+body {
+  background-color: aliceblue;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+#login-window {
+  position: relative;
+  background-color: antiquewhite;
+  padding: 10px 20px;
+  width: 250px;
+}
+
+.h-centered {
+  margin: 0 auto; /* center horizontally */
+}
+
+#login-window-title {
+  text-align: center;
+  font-size: 24px;
+  font-weight: bold;
+}
+
+#create-new-account {
+  width: 250px;
+  color: blue;
+  text-decoration: underline;
+  text-align: left;
+  cursor: pointer;
+}
+
+#login-window label {
+  font-weight: bold;
+}
+
+#login-window .lw-element {
+  margin: 10px 0px;
+}
+
+#login-window .input-wrapper {
+  border: 1px solid #ccc;
+  padding: 0px 4px;
+  background-color: white;
+  height: 25px;
+  border-radius: 5px;
+}
+
+#login-window input {
+  width: 100%;
+  height: 25px;
+  border: none;
+  font-size: 14px;
+  outline: none;
+  padding: 0px;
+}
+
+#login-window .lw-login {
+  margin: 20px 0px;
+  text-align: right;
+}
+
+.simpleapp-button {
```

Comment:
If you are going to have some general styles that are applicable all across your app the better thing to do is 'namespace' them. In this case I do not create a class called `button` but `simpleapp-button`